src/app/shared/services/principles.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@angular/core';
+
+import { IPrinciple, Principle } from '../principle.model';
+
+import principlesData from '../../../assets/tables/principles-data.json';
+import { Observable, of } from 'rxjs';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class PrinciplesService {
+    principles: IPrinciple[];
```

Comment:
this was only for testing right, so we should delete it?

Reply:
Forgot that was only for testing, removing and then going to merge the branch.